slackevents/parsers_test.go

```diff
@@ -17,6 +17,7 @@ func TestParserOuterCallBackEvent(t *testing.T) {
 				"event": {
 								"type": "app_mention",
 								"event_ts": "1234567890.123456",
+								"thread_ts": "1234567890.123456",
```

Comment:
`thread_ts` is not part of the app_mention event: https://api.slack.com/events/app_mention